Server/mods/deathmatch/logic/CMainConfig.h

```diff
@@ -227,4 +228,5 @@ class CMainConfig : public CXMLConfig
     int                        m_bFakeLagCommandEnabled;
     int                        m_iPlayerTriggeredEventIntervalMs;
     int                        m_iMaxPlayerTriggeredEventsPerInterval;
+    int                        m_checkResourceClientFiles;
```

Comment:
How does MTA handle the situation when `resource_client_file_checks` is missing in mtaserver.conf?

Reply:
It defaults to 1
https://github.com/multitheftauto/mtasa-blue/blob/e92702ca56a30b0c434cc29402b2d378ba4d7da4/Server/mods/deathmatch/logic/CMainConfig.cpp#L512-L520